agent/algorithm/logging.go

```diff
@@ -66,7 +67,7 @@ func (s *Stderr) Write(p []byte) (n int, err error) {
 		s.Logger.Error(string(buf[:n]))
 	}
 
-	if err := s.EventSvc.SendEvent("algorithm-run", "error", json.RawMessage{}); err != nil {
+	if err := s.EventSvc.SendEvent("AlgorithmRun", manager.Error.String(), json.RawMessage{}); err != nil {
```

Comment:
should be a string, also avoid imports from manager on agent

Reply:
That is `pkg/manager` and not `manager` package. Also, what do you mean by `should be a string`?